test/parallel/test-runner-cli.js

```diff
@@ -14,7 +14,7 @@ const testFixtures = fixtures.path('test-runner');
 
   assert.strictEqual(child.status, 1);
   assert.strictEqual(child.signal, null);
-  assert.strictEqual(child.stdout.toString(), '');
+  assert.strictEqual(child.stdout.toString(), 'TAP version 13\n');
```

Comment:
as we first start running and then check as we get the files asynchronously

Reply:
this should not be outputed if no test files are fond. we should emit the start event only at the first test